static/js/people.js

```diff
@@ -117,9 +124,7 @@ exports.is_known_user_id = function (user_id) {
 };
 
 function sort_numerically(user_ids) {
-    user_ids = _.map(user_ids, function (user_id) {
-        return parseInt(user_id, 10);
-    });
+    user_ids = map_int(user_ids);
```

Comment:
This conversion probably belongs in the caller.

Reply:
The story there was there were lots of callers, all passing in tiny list of user_ids.  I fixed it anyway.  In a couple cases, the caller already had ints, so no need to convert.  In the other cases, it wasn't completely clear that they were ints, so I convert them.